_config.yml

```diff
@@ -111,6 +111,11 @@ sponsors:
     image: /images/sponsors/bittele.svg
     logo_width: default
 
+  - name: Aries Robotics
```

Comment:
What defines the ordering of the sponsors on the homepage? Would it make sense for Aries to be first as they're the biggest now?